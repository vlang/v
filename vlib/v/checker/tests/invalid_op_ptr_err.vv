num := 10
p := &num
x := 5 * p
y := 5 / p
dump(x)
dump(p)
dump(y)
