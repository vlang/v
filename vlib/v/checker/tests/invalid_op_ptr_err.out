vlib/v/checker/tests/invalid_op_ptr_err.vv:3:6: error: infix `*` is not defined for pointer values
    1 | num := 10
    2 | p := &num
    3 | x := 5 * p
      |      ~~~~~
    4 | y := 5 / p
    5 | dump(x)
vlib/v/checker/tests/invalid_op_ptr_err.vv:4:6: error: infix `/` is not defined for pointer values
    2 | p := &num
    3 | x := 5 * p
    4 | y := 5 / p
      |      ~~~~~
    5 | dump(x)
    6 | dump(p)
